torch/functional.py

```diff
@@ -278,6 +279,50 @@ def isinf(tensor):
     return tensor.abs() == inf
 
 
+def meshgrid(*args, **kwargs):
```

Comment:
maybe rename `args` to `tensors`?

Reply:
Thank you for the suggestion, the name is now changed from args to tensors